### ISSUE TITLE
cannot read misc.xlsx

### ISSUE THREAD

**Issue by Tux** [jandrew/Spreadsheet-XLSX-Reader-LibXML#55]
```
$ perl -MSpreadsheet::XLSX::Reader::LibXML -wE'$a=Spreadsheet::XLSX::Reader::LibXML->new;$a->parse("files/misc.xlsx");say$a->error'
Can't call method "extractToFileHandle" on an undefined value at /pro/lib/perl5/site_perl/5.22.0/Spreadsheet/XLSX/Reader/LibXML.pm line 694.
```
I see nothing special with that file (https://github.com/Tux/Spreadsheet-Read/blob/master/files/misc.xlsx)

**Comment by Tux** [comment]
Same error for https://github.com/Tux/Spreadsheet-Read/blob/master/files/perc.xlsx

**Comment by jandrew** [comment]
This appears to have been fixed along with #51
